lib/paperclip/paperclip_processors/transcoder.rb

```diff
@@ -56,6 +56,18 @@ def make
         if output_is_image?
           @time = @time.call(@meta, @options) if @time.respond_to?(:call)
           @cli.filter_seek @time
+
+          if @auto_rotate && !@meta[:rotate].nil?
+            log "Adding rotation #{@meta[:rotate]}"
+            case @meta[:rotate]
+            when 90
+              @convert_options[:output][:vf] = "'transpose=1'"
```

Comment:
You don't have to. If the parameters are different, I guess it's better to move this filter to `av` gem instead of handling it in `paperclip-av-transcoder`.